<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasControl</class>
 <widget class="QWidget" name="CanvasControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Canvas Controls</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="openImageButton">
       <property name="text">
        <string>Use Image...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Shapes</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="shapeGridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="circleRadioButton">
         <property name="text">
          <string>Circle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rectRadioButton">
         <property name="text">
          <string>Rectangle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="lineRadioButton">
         <property name="text">
          <string>Line</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="triangleRadioButton">
         <property name="text">
          <string>Triangle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="textRadioButton">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Parameters</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QGridLayout" name="parametersLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="targetLabel">
         <property name="text">
          <string>Target</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="fontLabel">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="target2YSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="originXSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="targetXSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="textLineEdit"/>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="fillCheckBox">
         <property name="text">
          <string>Fill</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="textLabel">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="originLabel">
         <property name="text">
          <string>Origin</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="drawButton">
         <property name="text">
          <string>Draw</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="originYSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="eraseButton">
         <property name="text">
          <string>Clear...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="fontComboBox"/>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="targetYSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="target2XSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="target2Label">
         <property name="text">
          <string>Point 3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="colorLabel">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="selectColorButton">
         <property name="text">
          <string>Select Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>circleRadioButton</tabstop>
  <tabstop>lineRadioButton</tabstop>
  <tabstop>rectRadioButton</tabstop>
  <tabstop>textRadioButton</tabstop>
  <tabstop>triangleRadioButton</tabstop>
  <tabstop>originXSpinBox</tabstop>
  <tabstop>originYSpinBox</tabstop>
  <tabstop>targetXSpinBox</tabstop>
  <tabstop>targetYSpinBox</tabstop>
  <tabstop>target2XSpinBox</tabstop>
  <tabstop>target2YSpinBox</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>textLineEdit</tabstop>
  <tabstop>fillCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
